joomla/joomla-cms
# Fix options in mailtemplate form

Pull Request for Issue # .

### Summary of Changes
Fix a copy paste error in template form


### Testing Instructions
Code inspect - or apply the patch and make sure that tabs in mail templates are still correct 
![grafik](https://user-images.githubusercontent.com/1035262/165725982-602a0628-05ab-4a3c-a564-b1651544dbfc.png)
